src/DerConverter/Asn/DerAsnType.cs

```diff
@@ -40,6 +40,14 @@ internal static DerAsnType Parse(Queue<byte> data)
             var typeDataLength = data.DequeueDerLength();
             var typeData = new Queue<byte>(data.Dequeue(typeDataLength));
 
+            // Higher bits indicate the tag is context specific; lower bits indicate the tag value.
+            // E.g. "0xA0" => Context Specific (0),
+            //      "0xA3" => Context Specific (3)
+            if ((typeTag & DerAsnTypeTag.ContextSpecific) == DerAsnTypeTag.ContextSpecific)
```

Comment:
Do you have any link to documentation that describes this behavior? I wonder if we always need to chop-off the 2 MSb's.

Reply:
Yes, see https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-X.690-201508-I!!PDF-E&type=items , paragraph 8.1.2.2. Bits 6, 7 and 8 are not part of the tag number.